warehouse/ingest-core/src/test/java/datawave/iterators/PropogatingIteratorTest.java

```diff
@@ -214,6 +221,95 @@ public void testAggregateFour() throws IOException {
         
     }
     
+    @Test
+    public void testAggregateWithMaxUids() throws IOException {
+        TreeMultimap<Key,Value> map = TreeMultimap.create();
+        
+        map.put(newKey(SHARD, FIELD_TO_AGGREGATE, "abc"), new Value(createValueWithUid("abc.1").build().toByteArray()));
+        map.put(newKey(SHARD, FIELD_TO_AGGREGATE, "abc"), new Value(createValueWithUid("abc.2").build().toByteArray()));
+        map.put(newKey(SHARD, FIELD_TO_AGGREGATE, "abc"), new Value(createValueWithUid("abc.3").build().toByteArray()));
+        map.put(newKey(SHARD, FIELD_TO_AGGREGATE, "abc"), new Value(createValueWithUid("abc.4").build().toByteArray()));
+        
+        map.put(newKey(SHARD, FIELD_TO_AGGREGATE, "abd"), new Value(createValueWithUid("abc.3").build().toByteArray()));
```

Comment:
For clarity this value/uid could change to `xyz`. At first glace this looks like either `abd` or `abc.3` is a copy/paste typo